scene/gui/control.cpp

```diff
@@ -491,7 +491,7 @@ void Control::_validate_property(PropertyInfo &property) const {
 	} else if (Object::cast_to<Container>(parent_node)) {
 		// If the parent is a container, display only container-related properties.
 		if (property.name.begins_with("anchor_") || property.name.begins_with("offset_") || property.name.begins_with("grow_") || property.name == "anchors_preset" ||
-				(property.name.begins_with("rect_") && property.name != "rect_min_size" && property.name != "rect_clip_content" && property.name != "rect_global_position")) {
+				(property.name != "minimum_size" && property.name != "clip_contents" && property.name != "global_position")) {
```

Comment:
These lines are not equivalent, it used to say "all rect_*, except those 3", but now it needs to explicitly list all ex-rect properties except those 3.

Reply:
I've updated it to specify the other five instead using `||` and `==`